iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergMigration.java

```diff
@@ -266,10 +292,14 @@ public void testMigrationFailsForManagedTable() {
             "('storage_handler'='org.apache.iceberg.mr.hive.HiveIcebergStorageHandler')"));
   }
 
-  private void validateMigration(String tableName) throws TException, InterruptedException {
+  private Table validateMigration(String tableName, String tblProperties)
```

Comment:
can we keep  `validateMigration` with 1 argument, to avoid passing `null` in multiple places

Reply:
Done